sdcm/kafka/kafka_cluster.py

```diff
@@ -0,0 +1,171 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright (c) 2023 ScyllaDB
+import logging
+from pathlib import Path
+
+import requests
+
+from sdcm import cluster
+from sdcm.wait import wait_for
+from sdcm.remote import LOCALRUNNER
+from sdcm.utils.git import clone_repo
+from sdcm.utils.common import get_sct_root_path
+from sdcm.utils.remote_logger import DockerComposeLogger
+from sdcm.kafka.kafka_config import SctKafkaConfiguration
+
+# TODO: write/think more about the consumers
+
+LOGGER = logging.getLogger(__name__)
+
+
+class LocalKafkaCluster(cluster.BaseCluster):
+    def __init__(self, remoter=LOCALRUNNER):
```

Comment:
will sct_runner survive high load on kafka?

Reply:
It might, we can scale the runner as needed.

The idea is to have a setup that can work completely locally with docker backed for development

The next stage is building a cluster of kafka instead of the docker compose setup, and then some Kafka SaaS.

So for this initial step we can about building functionally, not yet about scale. scaling would be tested on VMs or on SaaS.

Reply:
worth adding a note that it's on sct runner and its size should be increased.
Generally small docstring would be great.